lib/elixir/lib/base.ex

```diff
@@ -397,7 +400,7 @@ defmodule Base do
       {:ok, "foob"}
 
   """
-  @spec decode64(binary, keyword) :: {:ok, binary} | :error
+  @spec decode64(binary, ignore: :whitespace, padding: boolean) :: {:ok, binary} | :error
```

Comment:
Is it valid to use such notation to cover reversed order of arguments `decode("...", padding: true, ignore: :whitespace)`?

Reply:
Great point. I've opened a PR to handle that situation.